autogen.sh

```diff
@@ -48,4 +48,11 @@ gtkdocize || exit $?
 autoreconf -fi || exit $?
 
 cd "$olddir"
-test -n "$NOCONFIGURE" || "$srcdir/configure" "$@"
+
+# If no arguments passed, will use default arguments for eos-metrics.
+if [ $# -eq 0 ]
+    then
+        test -n "$NOCONFIGURE" || "$srcdir/configure" "--enable-strict-flags" "--enable-gtk-doc" "--enable-gir-doc"
```

Comment:
Bahahaha, simultaneous comments.  Thanks, I agree with your logic(s), and will make that change.